File: 70-climbing-stairs/70-climbing-stairs.py
class Solution(object):
    def climbStairs(self, N):
        """
        :type n: int
        :rtype: int
        """
        
        steps=[None]*(N+1)
        
        steps[0]=1
        steps[1]=1
        
        for i in range(2,N+1):
            steps[i]=steps[i-1]+steps[i-2]
            
        return steps[N]
        
        
        

File: 70-climbing-stairs/70-climbing-stairs.cpp
class Solution {
public:
    int climbStairs(int n) {
        unsigned long long int one=1,two=1,temp;
        for(int i=0;i<n;i++)
        {   
            temp=one;
            one=one+two;
            two=temp;

        }
        return two;
        
    }
};